scripts/installs/install_rails_service.bat

```diff
@@ -0,0 +1,3 @@
+copy C:\Vagrant\resources\rails_server\start_rails_server.bat "C:\Program Files\Rails_Server"
+schtasks /create /tn "rails" /tr "\"cmd.exe\" /c \"C:\Program Files\Rails_Server\start_rails_server.bat\"" /sc onstart /NP /ru "SYSTEM"
```

Comment:
Is there any reason why we can't run rails as a service like we did with some of the other vulnerabilities?